### PR TITLE
Fix receivers of join email (fixes #524)

### LINKED ISSUE #524: Change receiver of offer join email
__Description:__

Change receiver from interested user to offer creator (Zgłoszenie chęci pomocy w ofercie)

### PR DESCRIPTION
**Pull request by magul**
__Story / Bug id:__ https://github.com/CodeForPoznan/volontulo/issues/524
__Description:__

After joining offer email is sent to organisation's owners instead of user that joined offer.

__New imports / dependencies:__

* pdbpp

__What tests do I need to run to validate this change:__

Join offer and look who got an email about it.

### CODE REVIEW

File: apps/volontulo/views/offers.py [hunk 1]
```diff
@@ -483,13 +483,12 @@ def post(request, slug, id_):  # pylint: disable=unused-argument
...
             offer.save()
-
             send_mail(
                 request,
                 'offer_application',
                 [
-                    user.email,
-                    request.POST.get('email'),
+                    up.user.email
+                    for up in offer.organization.userprofiles.all()
```
Comment (riv2q) [inline apps/volontulo/views/offers.py]: I prefer not to use shortcuts as a name of variables this improve readability in code, suggest to change this variable name from ``` up ``` to ``` user_profile ```

**Comment by magul** [comment]
@riv2q done

**Comment by magul** [comment]
@riv2q is there antything to do here?

**Comment by riv2q** [comment]
nope, looks ok now, merging. good work, keep going! :)

**Pull request merged**